streamx-flink/streamx-flink-shims/streamx-flink-shims-base/src/main/scala/com/streamxhub/streamx/flink/core/FlinkSqlExecutor.scala

```diff
@@ -58,7 +58,7 @@ object FlinkSqlExecutor extends Logger {
     }
 
     val configOptions = new JavaHashMap[String, ConfigOption[_]]
-    val configList = List(
+    val configList = scala.collection.immutable.List(
```

Comment:
why change here like this?

Reply:
Idea prompted me to add the prefix. Now I have removed the prefix and found no warning. I changed this back.